README.md

```diff
@@ -176,6 +192,69 @@ The Realm Flutter package name is `realm`
     realm.close();
     ```
 
+## Full contents of `catalog.dart`
+
+```dart
+import 'package:realm/realm.dart';
+
+part 'catalog.g.dart';
+
+@RealmModel()
+class _Item {
+    @PrimaryKey()
+    late int id;
+
+    late String name;
+    
+    int price = 42;
+}
+
+// Create a Configuration object
+var config = Configuration([Item.schema]);
+
+// Opean a Realm
+realm = Realm(config);
+
+var myItem = Item(0, 'Item', price: 4);
+
+// Open a write transaction
+realm.write(() {
+    realm.add(myItem);
+    var item = realm.add(Item(1, 'Item')..price = 20);
```

Comment:
```suggestion
    var item = realm.add(Item(1, 'Pencil')..price = 20);
```

Reply:
I added these changes in items names in order to have matches in filter when query is called.